main.css

```diff
@@ -0,0 +1,29 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300|Josefin+Slab|Arvo|Lato|Vollkorn|Abril+Fatface|Old+Standard+TT|Droid+Sans|Lobster|Inconsolata|Montserrat|Playfair+Display|Karla|Alegreya|Libre+Baskerville|Merriweather|Lora|Archivo+Narrow|Neuton|Signika|Questrial|Fjalla+One|Bitter|Varela+Round);
```

Comment:
Whoa cowboy! That's a LOT of fonts. Normally we try to stick to just one or two; each load slows down the page load time. Do we need all of these for any reason? It doesn't look like they're ever really used.